docusaurus/docs/operate/upgrades/3_module_auth.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Module Authorizations
+sidebar_position: 3
+---
+
+## Module Authorizations <!-- omit in toc -->
+
+Pocket Network utilizes an offchain governance mechanism that enables the community to vote on proposals.
+
+Once a proposal passes, or a decision by PNF is made on behalf of the DAO, PNF adjust the parameters necessary for the protocol's operation.
+
+- [Access Control](#access-control)
+  - [MainNet Authorizations](#mainnet-authorizations)
+    - [`x/gov` Module Granter](#xgov-module-granter)
+    - [`PNF` Account Grantee](#pnf-account-grantee)
+- [Examples](#examples)
+  - [Example: Adding a new Authorization](#example-adding-a-new-authorization)
+
+## Access Control
+
+### MainNet Authorizations
+
+The list of authorizations enabled on MainNet genesis can be found at [pokt-network/pocket-network-genesis/tree/master/shannon/mainnet](https://github.com/pokt-network/pocket-network-genesis/tree/master/shannon/mainnet).
+
+#### `x/gov` Module Granter
+
+The `x/gov` module granter is tied to address `pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t`.
```

Comment:
```suggestion
The `x/gov` module account is deterministically tied to address `pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t`. This will be true for any "pocket" network; e.g., LocalNet, TestNet, MainNet, etc.
```

Not sure how much detail you think is relevant but in case you think it's worth clarifying the relevance of the `x/gov` module account - why is it part of this picture? - may I suggest the following additional context for your consideration:

The `x/gov` module account is the default configured "authority" for all cosmos-sdk modules. Each module references its own configured "authority" when executing messages which require authorization (e.g. `MsgUpdateParams` messages).